src/col_tests/utils/test_settings.py

```diff
@@ -0,0 +1,42 @@
+def get_test_settings(test_type):
+    """Returns basic parameter settings for either instantaneous
+    or overdosing test calculations.
+
+    Args
+        test_type:      Str. Either 'instantaneous' or 'overdosing'
+
+    Returns
+        param_settings: Dict. Dictionary of parameter values for
+                        specified test type.
+    """
+
+    # TODO: Can we populate this directly from the spreadsheet instead of hardcoding?
+
+    if (test_type == 'instantaneous'):
+        param_settings = dict(
+            col_name = 'pH',
+            xlabel = 'pH (-)',
+            percent_factor = 100,
+            od_factor = {
+                'A': 1,
+                'B': 1,
+                'C': 1,
+                'D': 1,
+                'E': 1,
+            },
+        )
+    else:
+        param_settings = dict(
+            col_name = 'Lime_added_mg/l',
+            xlabel = 'Lime added (mg/l)',
+            percent_factor = 1,
+            od_factor = {
+                'A': 10,
+                'B': 20,
+                'C': 35,
+                'D': 50,
+                'E': 85,
+            },
+        )
+
+    return param_settings
```

Comment:
I think that after making some of the changes suggested above, `percent_factor` can be removed (it should always be 100) and `od_factor` should be renamed `dose` or `lime_dose`, or similar. This function should then describe the "real" differences between the two tests:

- For **instantaneous dissolution**, the dose of lime, $D$, is always 10 mg/l and the pH is varied
- For **overdosing**, the dose varies (`[10, 20, 35, 50, 85]` mg/l) and the pH is fixed at 4.6

The calculations should then be identical for all columns, with only $D$ and $pH$ changing (for a given lime product i.e. fixed $F_{Ca}$ and $F_{Mg}$). 

For the overdosing test, we then perform one additional calculation using only the data for columns where dose was varied and pH was fixed at 4.6. This is the calculation you have already implemented in `get_test_results`, so I think everything else should be the same

Reply:
Done!